src/backend/logger/__tests__/logfile.test.ts

```diff
@@ -13,8 +14,9 @@ jest.mock('../logger')
 jest.unmock('../logfile')
 
 let tmpDir = {} as DirResult
+const skipIfOnWin = platform() === 'win32' ? describe.skip : describe
 
-describe('logger/logfile.ts', () => {
+skipIfOnWin('logger/logfile.ts', () => {
```

Comment:
I wonder if it's better to do:

```
describe(..... () => {
  if (platform() === 'win32') {
    console.log('ignored on windows')
    return
  }

  ....
}
```

maybe it's just me though

Reply:
yeah this will make it easier to find the tests that are being skipped. I'll add this